src/components/TaskCard/index.module.scss

```diff
@@ -12,16 +12,54 @@
   border-radius: 0.313rem;
   border: 0.063rem solid $light-blue;
   height: 10.188rem;
+  position: relative;
   width: 23.875rem;
 }
 
+.task {
+  color: $navy-blue;
+  font-size: 1.5rem;
+  font-weight: 700;
+  line-height: 1.758rem;
+}
+
+.taskDone {
```

Comment:
please follow BEM structure for classnames. here the classname should be `task__done`